content/chapter-03/Intro.md

```diff
@@ -10,6 +10,5 @@
 * Lab: Instance types and and IO considerations
 * [Intermittent connectivity issues](./IntermittentConnectivity.md)
 * Lab: What to do when I cannot connect?
-* Caching and Session Management
-* Resilience of software design
+* [Resilience of software design](Resilience.md)
```

Comment:
Resilience of software design => Resilient Software Design

Reply:
Done